modules/nf-core/fastme/tests/main.nf.test

```diff
@@ -0,0 +1,128 @@
+                }
+            }
+        }
+
+        when {
+            process {
+                """
+                input[0] = TCOFFEE_SEQREFORMAT.out.formatted_file.collect{ meta, aln -> aln }
+                input[1] = []
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+    }
+
+    test("setoxin - phylip - with_tree") {
+
+        config "./main.config"
+
+        setup {
+            run("TCOFFEE_SEQREFORMAT") {
+                script "../../tcoffee/seqreformat/main.nf"
+                process {
+                    """
+                    input[0] = [ [ id: "test" ],
+                        file("https://raw.githubusercontent.com/nf-core/test-datasets/multiplesequencealign/testdata/setoxin.ref", checkIfExists: true)
+                    ]
+                    """
+                }
+            }
+            run("FAMSA_GUIDETREE") {
+                script "../../famsa/guidetree/main.nf"
+                process {
+                    """
+                    input[0] = [ [ id: "test" ],
+                                file("https://raw.githubusercontent.com/nf-core/test-datasets/multiplesequencealign/testdata/setoxin.ref", checkIfExists: true)
+                            ]
+
+                    """
+                }
+            }
+        }
+
+        when {
+            process {
+                """
+                input[0] = TCOFFEE_SEQREFORMAT.out.formatted_file.collect{ meta, aln -> aln }
+                input[1] = FAMSA_GUIDETREE.out.tree.collect{ meta, tree -> tree }
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+
+    }
+
+    test("setoxin - phylip - with_optional_output") {
+
+        config "./optionals.config"
+
+        setup {
+            run("TCOFFEE_SEQREFORMAT") {
+                script "../../tcoffee/seqreformat/main.nf"
+                process {
+                    """
+                    input[0] = [ [ id: "test" ],
+                        file("https://raw.githubusercontent.com/nf-core/test-datasets/multiplesequencealign/testdata/setoxin.ref", checkIfExists: true)
+                    ]
+                    """
+                }
+            }
+        }
+
+        when {
+            process {
+                """
+                input[0] = TCOFFEE_SEQREFORMAT.out.formatted_file.collect{ meta, aln -> aln }
+                input[1] = []
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.nwk) },
+                { assert snapshot(process.out.matrix) },
+                { assert snapshot(process.out.bootstrap) }
```

Comment:
As I understand, the bootstrap result is not deterministic so should not be snapshotted, and without calling .match() it shouldn't even generate a snapshot I don't think.
I suggest just checking for the files manually, or snapshot-matching the list of files in the output dir.